api.js

```diff
@@ -1,17 +1,12 @@
-import reducer from './lib/reducers';
-import updateStore from './lib/sync/update-store';
-import updateStorageManager from './lib/middleware/update-storage-manager';
-import makeBrainstemType from './lib/types/make-brainstem-type';
-import modelActions from './lib/actions/model';
-import collectionActions from './lib/actions/collection';
-import stopUpdateStore from './lib/sync/stop-update-store';
-
-export default {
-  reducer,
-  updateStore,
-  updateStorageManager,
-  makeBrainstemType,
-  modelActions,
-  collectionActions,
-  stopUpdateStore,
-};
+export { default as reducer } from './lib/reducers';
```

Comment:
This reminded me to write this in my post mortem es6 PR:

>  For spying on `default`, I've found you can do `import * as foo from './file-exporting-default';` and then do `spyOn(foo, 'default')` as long as it wasn't "re-exported",
as [re-exporting](http://2ality.com/2014/09/es6-modules-final.html#re-exporting) will definitely freeze it as a getter, but I've found if it's a direct export (not re-export) the spy can attach itself. This workaround only likely works because we're setting
babel to compile down to CommonJS (as, technically, any export should be a frozen live view)


I think what you're doing to test these here is:
https://github.com/mavenlink/brainstem-redux/pull/68/files#diff-46fc8f01898b4a269796c9653c9b2915R9

Which is probably fine. The only re-exporting issue I found was when we need to spyOn or td.replace. I'm going to assert that if we're doing that for one of these all the way on the bigmaven side, that's a test smell anyway, so we're probably fine. As long as you're confident we don't need to spy on these :)

Reply:
Yup. Appreciate the tip though. We will see if we need these. I will not merge this PR until bigmaven is green with the latest set of changes.